charts/foundry/values.yaml

```diff
@@ -54,10 +56,6 @@ securityContext:
     drop:
     - ALL
   readOnlyRootFilesystem: false
-  runAsNonRoot: true
-  # foundry user/group
-  runAsUser: 1000
-  runAsGroup: 1000
 
```

Comment:
try adding `fsGroup: 1000` here I think that should help to make the volume writable from non root user

Reply:
you can find more info here: https://kubernetes.io/docs/tasks/configure-pod-container/security-context/#configure-volume-permission-and-ownership-change-policy-for-pods